.github/dependabot.yml

```
@@ -105,3 +107,23 @@ updates:
         update-types:
           ['version-update:semver-major', 'version-update:semver-minor']
       - dependency-name: 'zone.js'
+
+  - package-ecosystem: npm
+    directory: '/interfaces/Portalicous'
+    schedule:
+      interval: weekly
+      day: tuesday
+    open-pull-requests-limit: 5
+    versioning-strategy: increase
+    groups:
+      development:
+        patterns:
+          - '@types/*'
+          - '*eslint*'
+          - '*prettier*'
+          - 'karma*'
```

I've been wondering whether we could get away with just grouping everything to avoid having to approve/merge/rebase so many PRs.

For example in my projects I [usually do](https://github.com/aberonni/improvdb/blob/main/.github/dependabot.yml#L13-L15):

```suggestion
    groups:
      dependencies:
        patterns:
          - "*"
```

If not that, then maybe we could at least add a catch-all group
```suggestion
    groups:
      development:
        patterns:
          - '@types/*'
          - '*eslint*'
          - '*prettier*'
          - 'karma*'
          - '*'
```

---

We could try it for a while and see how annoying/convenient it is..

The main separation for me is on the "can blindly merge if tests are green"-things (like the `eslint-*`-stuff) and the "Angular stuff needs to be updated together manually anyway"-stuff, etc.

I have created/used a specific workflow for those Angular updates: https://github.com/rodekruis/helpful-information/blob/main/.github/workflows/upgrade-dependencies.yml
But its not a silver bullet...

---

There will be some issues with some packages that have a different understanding of "breaking changes in a minor version" and with their matching peer-dependencies...

But starting with `*` and only adding the exceptions-when-necessary seems like a good approach. 👍